app/models/patient.rb

```diff
@@ -117,35 +117,9 @@ def latest_transfer
     where(purged: false)
       .where(pause_notifications: false)
       .where('patients.id = patients.responder_id')
-      .where.not('latest_assessment_at >= ?', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
-      .or(
-        where(purged: false)
-          .where(pause_notifications: false)
-          .where('patients.id = patients.responder_id')
-          .where(latest_assessment_at: nil)
-      )
-      .distinct
-  }
-
-  # Patients who are eligible for reminders:
-  #   - not purged AND
-  #   - notifications not paused AND
-  #   - valid preferred contact method AND
-  #   - HoH or not in a household AND
-  #   - we haven't sent them an assessment within the past 12 hours AND
-  #   - they haven't completed an assessment today OR they haven't completed an assessment at all
-  #   - (TODO in this scope rather than send_assessment) actively monitored OR has dependents that are being actively monitored
-  #
-  # NOTE: This method is currently being tested to be swapped in as the new reminder_eligible scope.
-  #       Once swapped in, the send_assessment method can be cut down to remove redundant logic.
-  scope :optimal_reminder_eligible, lambda {
-    where(purged: false)
-      .where(pause_notifications: false)
       .where.not(preferred_contact_method: ['Unknown', 'Opt-out', '', nil])
-      .where('patients.id = patients.responder_id')
       .where('last_assessment_reminder_sent <= ? OR last_assessment_reminder_sent IS NULL', 12.hours.ago)
-      .where('latest_assessment_at < ? OR latest_assessment_at IS NULL', Time.now.getlocal('-04:00').beginning_of_day)
-      .distinct
+      .where('latest_assessment_at < ? OR latest_assessment_at IS NULL', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
```

Comment:
I see that `.distinct` was removed. Was there a specific reason for this change?

Reply:
No good catch, though I don't actually think it's necessary because we're not going to get duplicate patients. I will add back for now to air on the side of caution.

Reply:
No good catch, though I don't actually think it's necessary because we're not going to get duplicate patients. I will add back for now to air on the side of caution.

Reply:
Squashed commits, but I added this back in the most recent push.